src/main/resources/templates/quoridor.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Welcome</title>
+    <link rel="stylesheet" type="text/css" th:href="@{/quoridor.css}" />
+</head>
+
+<body>
+    <div class="board">
+        <div th:each="square : ${squares}" class="square" th:style="${'top: ' + square.position.y * 40 + 'px; left: ' + square.position.x * 40 + 'px'}"></div>
+        <div class="pawn" th:style="${'top: ' + pawn.position.y * 40 + 'px; left: ' + pawn.position.x * 40 + 'px'}">
```

Comment:
Indentation issue

Reply:
Not sure what templating engine you're using, and if you can do my suggestion but I would initialize top and left variable on their own object before giving them to the style property, it would be much easier to read.